pyk/src/pyk/utils.py

```diff
@@ -499,6 +499,57 @@ def run_process_2(
     return res
 
 
+def run_proof_hint_process(
```

Comment:
Please move this to `krun.py`, and make it private.

Reply:
Done